x/records/keeper/epoch_unbonding_record.go

```diff
@@ -114,3 +117,24 @@ func (k Keeper) AddHostZoneToEpochUnbondingRecord(ctx sdk.Context, epochNumber u
 	}
 	return &epochUnbondingRecord, true
 }
+
+// TODO: unittest
+// TODO: what should this return?
+func (k Keeper) SetHostZoneUnbondings(ctx sdk.Context, zone stakeibctypes.HostZone, epochUnbondingRecordIds []uint64, status types.HostZoneUnbonding_Status) {
```

Comment:
What is the most likely case of failure here? Seems it'd only fail if something was very off (i.e. it was passed an invalid epoch unbonding record id) - in which case we'd probably want to exit immediately, rather than continue looping through. 

If you agree with that, this should probably return an error which is checked in the calling function. And then the calling function can propagate the error if it's dealing with only one host zone, or just `continue` if it's looping host zones.

If you think there's a common error case here that doesn't necessarily indicate that something is off, and we just want to skip over it - then I think it's fine to just return nothing and take no extra action in the calling function.

Reply:
Think that probably makes sense, updated to return an error